src/net/sourceforge/kolmafia/textui/command/MayamCommand.java

```diff
@@ -0,0 +1,103 @@
+package net.sourceforge.kolmafia.textui.command;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import net.sourceforge.kolmafia.KoLConstants.MafiaState;
+import net.sourceforge.kolmafia.KoLmafia;
+import net.sourceforge.kolmafia.RequestThread;
+import net.sourceforge.kolmafia.objectpool.ItemPool;
+import net.sourceforge.kolmafia.preferences.Preferences;
+import net.sourceforge.kolmafia.request.GenericRequest;
+import net.sourceforge.kolmafia.session.InventoryManager;
+
+public class MayamCommand extends AbstractCommand {
+  private static final List<List<String>> SYMBOL_POSITIONS =
+      List.of(
+          List.of("yam", "clock", "explosion"),
+          List.of("yam", "eyepatch", "cheese", "wall"),
+          List.of("yam", "lightning", "bottle", "board", "meat"),
```

Comment:
the image name is board.png, but the hover text is "Wood".

I think this one should be "wood" instead of "board".

Reply:
A javascript var on the page keys it to "board", but the value is "Wood":

`var rings = [{"yam":"Yam","clock":"Clock","explosion":"Explosion"},{"yam":"Yam","eyepatch":"Eyepatch","cheese":"Cheese","wall":"Wall"},{"yam":"Yam","lightning":"Lightning","bottle":"Bottle","board":"Wood","meat":"Meat"},{"yam":"Yam","sword":"Sword","vessel":"Vessel","fur":"Fur","chair":"Chair","eye":"Eye"}];`

But then everything the user sees calls it "wood" except the image name, so would "wood" probably be better.

Reply:
I used that var to build this list, yeah. But I can change it to wood